cpp/src/utilities/cudf_utils.h

```diff
@@ -11,8 +11,8 @@
 #define CUDA_DEVICE_CALLABLE __device__ __forceinline__
 #define CUDA_LAUNCHABLE __global__
 #else
-#define CUDA_HOST_DEVICE_CALLABLE
-#define CUDA_DEVICE_CALLABLE
+#define CUDA_HOST_DEVICE_CALLABLE inline
```

Comment:
I had to make the same change in my DLpack PR. I also changed the `__forceinline__` above to `inline`.